sycl/test/deprecate_get_count.cpp

```diff
@@ -0,0 +1,12 @@
+// RUN: %clangxx -fsycl -sycl-std=2020 -Xclang -verify -Xclang -verify-ignore-unexpected=note %s -o %t.out
```

Comment:
I suggest using `-verify` with `-fsyntax-only` if possible.

Reply:
Moved test to sycl_2020_deprecations.cpp and removed this file